pkg/nmi/server/server.go

```diff
@@ -458,6 +463,11 @@ func (s *Server) defaultPathHandler(logger *log.Entry, w http.ResponseWriter, r
 	w.Write(body)
 }
 
+// forbiddenHandler responses to any request with HTTP 403 Forbidden
+func forbiddenHandler(w http.ResponseWriter, r *http.Request) {
+	http.Error(w, "", http.StatusForbidden)
```

Comment:
since NMI is the one blocking the requests when deployed, I'm thinking maybe we should have an error msg along with 403. 

@kkmsft thoughts?

Reply:
That's probably a good idea, in case someone gets confused and tries to contact Azure support.